lib/model_runner/mr_testing.ts

```diff
@@ -38,28 +41,27 @@ export class MRTestingConfig {
     public SM_VARIANT_NAME = "AllTraffic",
     public SM_CPU_INSTANCE_TYPE = "ml.m5.xlarge",
     public SM_GPU_INSTANCE_TYPE = "ml.p3.2xlarge",
-
     // bucket names
     public S3_RESULTS_BUCKET = "test-results",
     public S3_IMAGE_BUCKET = "test-images",
     // path to test images
     public S3_TEST_IMAGES_PATH = "assets/images",
-
     public MODEL_DEFAULT_CONTAINER = "awsosml/osml-models:main",
     // ecr repo names
     public ECR_MODEL_REPOSITORY = "model-container",
     // path to the control model source
     public ECR_MODELS_PATH = "lib/osml-models",
     // build target for control model container
-    public ECR_MODEL_TARGET = "osml_model"
+    public ECR_MODEL_TARGET = "osml_model",
+    public REPOSITORY_ACCESS_MODE = "Platform"
   ) {}
 }
 
 export interface MRTestingProps {
   // the osml account interface
   account: OSMLAccount;
-  // the model runner vpc
-  vpc: IVpc;
+  // the model runner osmlVpc
```

Comment:
"the model runner vpc" may have been okay here

Reply:
Yeah I can update that